src/main/java/com/telefonica/iot/perseo/Utils.java

```diff
@@ -274,6 +275,32 @@ public static String getBodyAsString(HttpServletRequest request) throws IOExcept
         return sb.toString();
     }
 
+    /**
+     * It implements a method that, following the same mechanism as putCorrelatorAndTrans, uses org.slf4j.MDC
+     * to store the necessary headers, which will be necessary for the request in pursuit of activation of
+     * an action. The same thread will use the MDC to extract this data.
+     * @param rule JSON with the rule information
```

Comment:
Style: blank line in JavaDoc just before `@param`

Reply:
Fixed in 589775e